robottelo/common/helpers.py

```diff
@@ -151,6 +151,32 @@ def generate_strings_list(len1=None):
     ]
 
 
+def generate_strings_dict(len1=None):
+    """Generates a dictionary of all the input strings.
+
+    :param int len1: Specifies the length of the strings to be
+        be generated. If the len1 is None then the dict is
+        returned with string types of random length.
+    :return: Returns a dict of various string types.
+    :rtype: dictionary
+
+    """
+    if len1 is None:
+        len1 = gen_integer(3, 30)
+    return {
+        str_type: gen_string(str_type, len1)
+        for str_type
+        in (u'alpha', u'numeric', u'alphanumeric',
+            u'latin1', u'utf8', u'cjk', u'html')
+    }
+
+
+def removekey(d, key):
+    r = dict(d)
+    del r[key]
+    return list(r.values())
```

Comment:
This method is only used in `test_gpgkey.py`, right? In that case, how about moving it to that module and doing something like this:

```python
def _remove_numeric(strings):
    if bz_bug_is_open(1184480):
        del strings['numeric']
    return strings.values()
```

This would allow you to drop the TODO from `test_gpgkey.py` and function would be placed in the one module which actually uses it, which are both good things.